### ISSUE TITLE
Play/Pause keyboard button does not work

### ISSUE THREAD

**Issue by fenryxo** [tiliado/nuvola-app-mixcloud#11]
@voxik commented on [Tue Sep 19 2017](https://github.com/tiliado/nuvolaruntime/issues/375)

Play/Pause keyboard button does not work anymore using Mixcloud app. This is the warning I see in my Journal:

~~~
Sep 19 09:04:12 localhost.localdomain eu.tiliado.NuvolaAppMixcloud.desktop[7979]: Runner: [WARNING Δ01m51s432452us DioriteGtk] Action.vala:74: Cannot activate action 'toggle-play', because it is disabled.
~~~

$ rpm -qa nuvola*
nuvolaruntime-4.7.0-1.fc28.x86_64
nuvola-app-mixcloud-3.2-3.fc28.noarch

$ rpm -q gtk3
gtk3-3.22.20-1.fc28.x86_64

**Comment by fenryxo** [comment]
Hello @s83. Could you look at it?

**Comment by s83** [comment]
Thank you @voxik for reporting that, I'll look into that ASAP.

**Comment by fenryxo** [comment]
Hello @s83. Is there any progress?

**Comment by fenryxo** [comment]
Given that the script is broken for at least two months, it has been practically orphaned and Nuvola 4.9 might drop it officially.

**Comment by voxik** [comment]
Just FTR, it seems that Mixcloud has completely changed the technology they are using for their UI :/

**Comment by fenryxo** [comment]
Fixed in release 4.0, which requires Nuvola 4.9.84 because of tiliado/nuvolaruntime@d3c2981. The flatpak package will be updated within a few days. Thank you for reporting this issue.
